keyboards/8pack/config.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x2171
+#define DEVICE_VER      0x0002
+#define MANUFACTURER    Charles Garcia
+#define PRODUCT         8-Pack
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCING_DELAY 5
```

Comment:
This definition has had its name changed
```suggestion
#define DEBOUNCE 5
```